src/main/java/team4/footwithme/chat/api/ChatMemberApi.java

```diff
@@ -1,12 +1,34 @@
 package team4.footwithme.chat.api;
 
+import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import team4.footwithme.chat.api.request.ChatMemberRequest;
+import team4.footwithme.chat.service.ChatMemberService;
+import team4.footwithme.global.api.ApiResponse;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping
+@RequestMapping("/api/v1")
 public class ChatMemberApi {
+    private final ChatMemberService chatMemberService;
+
+    /**
+     * 채팅방 초대
+     */
+    @PostMapping("/chat/member")
+    public ApiResponse<String> inviteChatMember(@RequestBody @Valid ChatMemberRequest chatMemberRequest) {
+        String message = chatMemberService.joinChatMember(chatMemberRequest.toServiceRequest());
+        return ApiResponse.created(message);
```

Comment:
```suggestion
        return ApiResponse.created(chatMemberService.joinChatMember(chatMemberRequest.toServiceRequest()));
```
저는 그냥 한줄에 넣어서 사용합니다 ㅎㅎ

Reply:
저도 생각했던 문제인데 인라인으로 만드는게 괜찮을거같네요